[v1.1.2]Add network relation explanation (#291)
harvester/docs: versioned_docs/version-v1.1/networking/harvester-network.md

To create a new untagged network, go to the **Networks > Networks** page and click the **Create** button. You have to specify the name, select the type `Untagged Network` and choose the cluster network.

![](/img/v1.1/networking/create-untagged-network.png)

:::note

> - Harvester supports updating and deleting VM network. You have to stop VMs which use the VM network before action execution.

Reviewer: ```suggestion
- Harvester supports updating and deleting VM networks. You have to stop VMs that use the VM network before action execution.
```

> - Starting from Harvester v1.1.2, Harvester supports updating and deleting VM networks. You have to stop VMs that use the VM network before action execution.

Reviewer 2: ```suggestion
Starting with Harvester v1.1.2, Harvester supports updating and deleting VM networks. Make sure to stop all affected VMs before updating or deleting VM networks.
```
